core/src/main/resources/lib/layout/header/searchbox.jelly

```diff
@@ -14,7 +14,7 @@
       </div>
     </j:set>
 
-    <button id="button-open-command-palette"
+    <button id="button-open-command-palette" aria-label="<%= l.jenkins.model.Messages.OpenSearch() %>"  data-label-key="jenkins.model.Messages.OpenSearch">
```

Comment:
I don't know what that is but that is not valid jelly syntax.
Just use `aria-label="${%Open Search}"` and remove the `data-label-key` attribute